test/utils.dart

```diff
@@ -107,7 +103,7 @@ class _MultipartBodyMatches extends Matcher {
 ///
 /// [message] can be a String or a [Matcher].
 Matcher isClientException([message]) => predicate((error) {
-      expect(error, new isInstanceOf<http.ClientException>());
+      expect(error, const isInstanceOf<http.ClientException>());
```

Comment:
We generally use `new isInstanceOf`, might as well keep this consistent.

Reply:
Any particular rules for when you want `const` used?

Reply:
I usually avoid it in general unless it's in a place where allocations may affect performance.